openstack/cerebro/templates/domain-ccadmin-seed.yaml

```diff
@@ -55,30 +55,64 @@ spec:
           description: Cerebro Jenkins domain.
         {{- end }}
       role_assignments:
+      - user: {{ .Values.technicalUserPerRegion }}
+        role: admin
+      - user: {{ .Values.technicalUserPerRegion }}
+        role: audit_viewer
+      - user: {{ .Values.technicalUserPerRegion }}
+        role: automation_admin
+      - user: {{ .Values.technicalUserPerRegion }}
+        role: compute_admin
+      - user: {{ .Values.technicalUserPerRegion }}
+        role: compute_admin_wsg
       - user: {{ .Values.technicalUserPerRegion }}
         role: compute_viewer
+      - user: {{ .Values.technicalUserPerRegion }}
+        role: dns_hostmaster
       - user: {{ .Values.technicalUserPerRegion }}
         role: dns_webmaster
+      - user: {{ .Values.technicalUserPerRegion }}
+        role: dns_zonemaster
+      - user: {{ .Values.technicalUserPerRegion }}
+        role: image_admin
+      - user: {{ .Values.technicalUserPerRegion }}
+        role: keymanager_admin
       - user: {{ .Values.technicalUserPerRegion }}
         role: kubernetes_admin
+      - user: {{ .Values.technicalUserPerRegion }}
+        role: masterdata_admin
       - user: {{ .Values.technicalUserPerRegion }}
         role: member
       - user: {{ .Values.technicalUserPerRegion }}
         role: monitoring_viewer
+      - user: {{ .Values.technicalUserPerRegion }}
+        role: network_admin
       - user: {{ .Values.technicalUserPerRegion }}
         role: network_viewer
       - user: {{ .Values.technicalUserPerRegion }}
         role: registry_admin
+      - user: {{ .Values.technicalUserPerRegion }}
+        role: registry_viewer
+      - user: {{ .Values.technicalUserPerRegion }}
+        role: resource_admin
       - user: {{ .Values.technicalUserPerRegion }}
         role: resource_viewer
+      - user: {{ .Values.technicalUserPerRegion }}
+        role: role_admin
       - user: {{ .Values.technicalUserPerRegion }}
         role: securitygroup_admin
+      - user: {{ .Values.technicalUserPerRegion }}
+        role: sharedfilesystem_admin
       - user: {{ .Values.technicalUserPerRegion }}
         role: sharedfilesystem_viewer
+      - user: {{ .Values.technicalUserPerRegion }}
+        role: objectstore_admin
+      - user: {{ .Values.technicalUserPerRegion }}
+        role: volume_admin
       - user: {{ .Values.technicalUserPerRegion }}
         role: volume_viewer
     groups:
-    - name: CCADMIN_CEREBRO_ADMIN
+    - name: CCADMIN_CEREBRO_TECHNICAL_TEAM_USERS

```

Comment:
Are you sure you don't want to use the profiles `CC Cerebro Admin` and `CC Cerebro Users` anymore?

Reply:
The following had been created by Rajiv:

------------------------------------------------------------------------
Domain Name:            CC CCADMIN
Profile Name:              CC CCADMIN CEREBRO User Team Account Management
Description:                 Get access to CEREBRO User Team account in the Converged Cloud CCADMIN OS Domain
 
Access Level Name:         CCADMIN_DOMAIN_USERS
Access Level Name:         CP_JUMP_CUSTOMER
Access Level Name:         CCADMIN_CEREBRO_TEAM_USERS
Access Description:        Members of CEREBRO and all sub-projects
 
Approvers:                 I316717, I053411, D059352, D059373, I547197, I508368
------------------------------------------------------------------------

------------------------------------------------------------------------
Domain Name:            CC CCADMIN
Profile Name:              CC CCADMIN CEREBRO Technical Team Account Management
Description:                 Get access to CEREBRO Technical Team account in the Converged Cloud CCADMIN OS Domain
 
Access Level Name:         CCADMIN_DOMAIN_USERS
Access Level Name:         CP_JUMP_CUSTOMER
Access Level Name:         CCADMIN_CEREBRO_TECHNICAL_TEAM_USERS
Access Description:        Members of CEREBRO and all sub-projects
 
Approvers:                 I316717, I053411, D059352, D059373, I547197, I508368
------------------------------------------------------------------------